gempython/tools/amc_user_functions_xhal.py

```diff
@@ -0,0 +1,343 @@
+import os, signal, sys
+sys.path.append('${GEM_PYTHON_PATH}')
+
+from ctypes import *
+from gempython.utils.gemlogger import colormsg
+from gempython.utils.nesteddict import nesteddict
+from gempython.utils.wrappers import envCheck
+from time import sleep
+
+from rw_reg import Node, parseInt, parseXML
```

Comment:
`rw_reg` is `xhal`/`reg_utils` now right?
suspect that as is, this will not work

Reply:
atm it is in xhal. But won't exist anymore after integration of reg_utils. Will be in:
`reg_utils.reg_interface.common.reg_xml_paser`
However it is fine to go as is right now and change later with inclusion of revised xhal/reg_utils

Reply:
yes at the moment all that is needed is to setup the `xhal` env.

When will integration of `reg_utils` be performed?  If it is a short time scale I can make this change now.  Otherwise I would prefer to leave it *as is* for the time being so a working release could be generated.

Reply:
if by "fine", you mean it won't currently work, i agree (dependence on removed `GEM_PYTHON_PATH`)

Reply:
`reg_utils` shoud be integrated next week. Actually, it is ready now barring the documentation completion in travis/docker what I fight right now.

Reply:
better to decouple deployment from documentation

Reply:
well, documentation is a sort of integration test here...

Reply:
> if by "fine", you mean it won't currently work, i agree (dependence on removed GEM_PYTHON_PATH)

I've tested this in cmsgemos `v0.3.1` it works fine...I don't follow.

Reply:
because while you test, `PYTHONPATH` was explicitly set to include the location of the `rw_reg.py` file